### PR TITLE
WIP: Install snapshot CRDs and controller from the external-snapshotter PR

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Manually modify prow.sh to install snapshot CRDs and controller from the PR.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by xing-yang** [comment]
@pohly can you take a look of this?  I modified prow.sh manually to install snapshot CRDs and controller from the PR.  If this works, I'll submit a PR to csi-release-tools.

File: release-tools/prow.sh [hunk 1]
```diff
@@ -733,8 +745,54 @@ install_snapshot_controller() {
...
+                  registry=$(echo "$image" | sed -e 's;\(.*\)/.*;\1;')
+                  name=$(echo "$image" | sed -e 's;.*/\([^:]*\).*;\1;')
+                  tag=$(echo "$image" | sed -e 's;.*:;;')
+
+	          # Variables are with underscores and upper case.
+                  varname=$(echo $name | tr - _ | tr a-z A-Z)
+
+	          # Now replace registry and/or tag
+	          NEW_TAG="csiprow"
+                  line="$(echo "$nocomments" | sed -e "s;$image;${name}:${NEW_TAG};")"
```
Comment (pohly) [inline release-tools/prow.sh:778]: Indention is inconsistent, doing it without tabs is preferred.

Comment (xing-yang) [reply release-tools/prow.sh:777]: Fixed

**Pull request closed without merging**